src/react/client/components/FAB/index.tsx

```
@@ -0,0 +1,59 @@
+    <>
+      <div className="shortcut-assistant-fab">
+        <button
+          className={`shortcut-assistant-fab-button ${isOpen ? 'open' : ''}`}
+          onClick={handleToggle}
+        >
+          <span className="shortcut-assistant-fab-icon">+</span>
+        </button>
+
+        {isOpen && (
+          <div className="shortcut-assistant-fab-menu">
+            <button className="shortcut-assistant-fab-menu-item">Analyze</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)